src/pages/new/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import ThreeParagraphBlock from "../../components/grid-aware/ThreeParagraphBlock";
+import image1 from "../../components/grid-aware/ThreeParagraphBlock/stories/image1.png";
+import image2 from "../../components/grid-aware/ThreeParagraphBlock/stories/image2.png";
+import image3 from "../../components/grid-aware/ThreeParagraphBlock/stories/image3.png";
+import Layout from "../../components/layout";
+
+export default () => (
+  <Layout>
+    <ThreeParagraphBlock
+      title="Get involved"
+      paragraph1={{
+        title: "Volunteer",
+        description:
+          "Volunteers make our work possible. There are several ways to support our mission. Learn more and get involved.",
+      }}
+      paragraph2={{
+        title: "Partnerships",
+        description:
+          "We work with companies, nonprofits, and local governments to empower the community. Reach out to us.",
+      }}
+      paragraph3={{
+        title: "Donate",
+        description:
+          "Our programs are largely funded by donations from people who care about bridging the digital divide. Support ShelterTech today.",
+      }}
+      image1={{
+        url: image1,
+        alt: "Two volunteers working on a laptop together at a datathon.",
+      }}
+      image2={{
+        url: image2,
+        alt: "Team posing for a photo after a design workshop.",
+      }}
+      image3={{
+        url: image3,
+        alt: "Multiple volunteers working at a datathon.",
+      }}
+      ctaTitle="Volunteer, donate, or reach out to our partnerships team"
+      ctaButtons={[
+        { text: "Volunteer" },
+        { text: "Donate" },
+        { text: "Work with us" },
+      ]}
```

Comment:
Button behaviors needed when clicked.

Reply:
Yeah, I was waiting for https://github.com/ShelterTechSF/sheltertech.org/pull/73 to merge in first, since that was going to change the APIs, but I can now swap these out. I'll still have to dummy these out to some extent, since the new pages/interactions don't exist yet, but perhaps I can create placeholders for the new pages as well.

Reply:
This is updated now. I created placeholder pages for the volunteer and donate pages in https://github.com/ShelterTechSF/sheltertech.org/pull/75/commits/ab685a04999584fd8432de58fefb7d23e94fe2de, and the first two links now link there. The third one I have just dummied out with an `onClick` that does nothing.